proposals/array_append.json

```
@@ -34,6 +46,11 @@
             }
         }
     },
+    "exceptions": {
+        "LabelExists": {
+            "message": "A label with the specified name exists."
```

suggestion

> An array item with specified label already exists

---

We don't use "item" anywhere else, instead we use value or element. So I'd propose:

```suggestion
            "message": "An array element with the specified label already exists."
```